src/js/background/context-menu.js

```diff
@@ -300,10 +355,15 @@ const relayContextMenus = {
       return relayAddresses;
       
     },
-    getGeneratedForHistory: async (website) => {
-      const { relayAddresses } = await browser.storage.local.get("relayAddresses");
-      
-      return relayAddresses.some(alias => website === alias.generated_for);
+    checkIfAnyMasksWereGeneratedOrUsedOnCurrentWebsite: async (website) => {
+      const relayAddresses = await relayContextMenus.utils.getAliases();
+
+      // Short circuit if not current site detected
+      if (website === null) {
+        return false;
+      }
+
+      return relayAddresses.some(alias => website === alias.generated_for || alias.used_on?.includes(website));
```

Comment:
Probably an edge case, but wouldn't this potentially match e.g. a mask used on `github.io`, even though the current user is at `othersite.github.io`? We miight want to do exact matches:

```suggestion
      return relayAddresses.some(alias => website === alias.generated_for || alias.used_on?.split(",").includes(website));
```

(Likewise for the other instances of `includes` below.)

Reply:
Per conversation on Slack with Tony: 

> Hey reviewing right now. and i think that's okay, can we try releasing it like this and if we get feedback that it's messing users up or causing confusion we can file a bug to change it?

Adding a comment to capture this bug for future us. Comment added in 146b305533e54ea8a749cab984c3f61f891ab852

Reply:
```
  // BUG: Using includes without spliting on ",", there can 
  // be some false positive results for entries that share the same root domain. 
  // eg: Github mask would show up on example.github.io
  // See full conversation: https://github.com/mozilla/fx-private-relay-add-on/pull/342#discussion_r897755698
```